docs/software/container-engine/resource-hook.md

```diff
@@ -169,6 +169,32 @@ The hook is activated by setting the `com.hooks.cxi.enabled` annotation, which
     4194304             23925.61
     ```
 
+!!! tip "How to check if the CXI provider is available"
+
+    You can check if the CXI provider is available using `fi_info`
+    (within the container):
```

Comment:
The `within the container` is the key context here (I think?)

Maybe start the section with something like "If you are uncertain about whether the CXI provider was configured correctly inside your running container..."